tribe-ext-recurring-cleanup/index.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar PRO Extension: Cleanup recurring events
```

Comment:
Title capitalization: would recommend `Cleanup Recurring Events` instead of current `Cleanup recurring events`